LICENSE.txt

```diff
@@ -0,0 +1,11 @@
+Copyright <YEAR> <COPYRIGHT HOLDER>
```

Comment:
I'm wondering if we need this? Can we link to the license in the main repo.

Reply:
Yeah, I think we should drop this file. Setting up a license file could be dependent on it's use.

Reply:
Removed the license file